webapp/portlet/src/main/webapp/vue-apps/space-settings/main.js

```diff
@@ -34,7 +33,6 @@ export function init(maxUploadSize) {
       template: `<space-settings
                   v-cacheable="{cacheId: '${cacheId}'}"
```

Comment:
```suggestion
```
No need to use `v-cacheable` anymore since this mechanism has been disabled by default.